street_lights/index.html

```diff
@@ -2,11 +2,48 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title>Światła drogowe</title>
+    <title>Swiatła drogowe</title>
   </head>
+<style>
+#streetLights {
+border-style : solid;
+border-width: 5px;
+width : 205px;
+height : 618px;
+}
+#red {
+color: red;
```

Comment:
```css
#red
#yellow
#green
```

Różnią się jedynie właściwością `color` czy nie uważasz że definiowanie całego element za każdym razem to przesada?

```
border-style : solid;
border-width: 3px;
border-radius: 50%;
width: 200px;
height: 200px; 
```

Jak bedziesz miał zmienić wysokość wszystkich świateł to bedziesz musial zmienić wszystkie definicje. Niepotrzebna robota.

Wyciągnij wspólne cześci do klasy `light`. Uzyj w `html` dla elementów.